types/gl-react-image/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for gl-react-image 3.2
+// Project: https://github.com/gre/gl-react-image
+// Definitions by: Martin Badin <https://github.com/martin-badin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+export interface GLImageProps {
+    center?: [number, number];
+    resizeMode: 'cover' | 'free' | 'contain' | 'stretch';
```

Comment:
From the docs and the source, it looks like `resizeMode` can be omitted and `'stretch'` will be used as a default.

Reply:
you are right but 'contain' will be used as a default.

https://github.com/gre/gl-react-image/blob/master/src/index.js#L163